x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/RestartIndexFollowingIT.java

```diff
@@ -151,4 +160,16 @@ private void cleanRemoteCluster() throws Exception {
             assertThat(infos.size(), equalTo(0));
         });
     }
+
+    private boolean hasFollowTaskWithFatalException(String indexName) {
```

Comment:
I'd prefer a more precise name here, we're only checking for the specific case of a disconnected cluster.

Reply:
I renamed to `isFollowerStoppedBecauseOfRemoteClusterDisconnection` but I'm not sure it is much better 😬